var/spack/repos/builtin/packages/gettext/package.py

```diff
@@ -30,6 +30,7 @@ class Gettext(AutotoolsPackage, GNUMirrorPackage):
     variant('libunistring', default=False, description='Use libunistring')
 
     depends_on('iconv')
+    depends_on('automake')
```

Comment:
```suggestion
    depends_on('automake', type='build', when='...')
```

The when clause depends on the output of `spack arch` on your system where it fails.

Reply:
updated it:
https://github.com/spack/spack/blob/8f7b6034ca818cd50c698ff2e918a5a182ae981d/var/spack/repos/builtin/packages/gettext/package.py#L33

I agree that this makes sense.  Did I do it correctly like that?